src/layouts/AppHeader.js

```diff
@@ -1,65 +1,119 @@
 import React from 'react';
-import { Layout, Menu } from 'antd';
+import { Icon, Layout, Menu } from 'antd';
 import { FormattedMessage } from 'react-intl';
 import { UserContext } from '../UserContext';
 import { Link } from "react-router-dom";
 import logo from '../assets/logo.png';
 import './AppHeader.less';
 
 const { Header } = Layout;
+const { SubMenu } = Menu;
 
 class AppHeader extends React.Component {
   render() {
     return (
       <UserContext.Consumer>
-        { ({switchLang}) => {
+        { ({switchLang, isLoggedIn}) => {
```

Comment:
FYI, This actualy have 3 states: loading, not loggedin, loggedin. 
Currently loading is treat as not loggedin in your code. You can hope that the loading is pretty fast. Maybe add a todo comment if you don't want to change it now.